repository.json

```diff
@@ -678,6 +678,34 @@
             "version": "1.9.7",
             "url": "https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-1.9.7-windows.zip",
             "fileMatchInside": ".*/phantomjs.exe$"
+        },
+        {
+            "name": "marionette",
+            "platform": "windows",
+            "bit": "32",
+            "version": "0.8.0",
+            "url": "https://github.com/mozilla/geckodriver/releases/download/v0.8.0/geckodriver-v0.8.0-win32.zip"
+        },
+        {
+            "name": "marionette",
+            "platform": "linux",
+            "bit": "64",
+            "version": "0.8.0",
+            "url": "https://github.com/mozilla/geckodriver/releases/download/v0.8.0/geckodriver-0.8.0-linux64.gz"
+        },
+        {
+            "name": "marionette",
+            "platform": "mac",
+            "bit": "32",
+            "version": "0.8.0",
+            "url": "https://github.com/mozilla/geckodriver/releases/download/v0.8.0/geckodriver-0.8.0-OSX.gz"
+        },
+        {
+            "name": "marionette",
+            "platform": "mac",
+            "bit": "64",
+            "version": "0.8.0",
+            "url": "https://github.com/mozilla/geckodriver/releases/download/v0.8.0/geckodriver-0.8.0-OSX.gz"
         }
```

Comment:
Best would be to follow the strategy @pumano has suggested:

driver 32bit --> 32bit-driver-url
driver 64bit --> 32bit-driver-url

This would remove the code smells. But we need tests and this should be done in an different PR.